README.md

```diff
@@ -107,7 +122,7 @@ The Core SDK includes classes that interact with the Immutable X APIs.
 import (
     "context"
     "fmt"
-    "github.com/immutable/imx-core-sdk-golang/generated/api"
+    "github.com/immutable/imx-core-sdk-golang/imx/api"
     "github.com/immutable/imx-core-sdk-golang/config"
```

Comment:
This import path and a few others like workflow no longer exist.
```
    "github.com/immutable/imx-core-sdk-golang/config"
```
Also, all the code samples need to update to the new way of calling everything from `imx` and accessing the methods from `Client` instead.

The samples should not use the `api` directly and should instead be calling everything from `Client` and related wrapper methods.

Reply:
We will not have standard API Requests in the docs anymore.
Will remove this.